src/migrate_parsetree_driver.ml

```diff
@@ -456,7 +469,7 @@ let print_transformations () =
   |> print_group "Registered Derivers"
 
 
-let run_as_standalone_driver argv =
+let run_as_standalone_driver exit_on_error argv =
```

Comment:
Just so nitpicking: given that a boolean argument is not super-descriptive on its own and could mean anything, it's usually better to label such arguments so that at the call site we know what they mean

Reply:
I agree, but this function is internal to the `.ml` file (at this point), and at its only call site, it's called with an argument with the same name.

Reply:
Agreed that it's clear right now, but code changes over time :) After a few people have worked on this module, the code might look pretty different. It's better to make things as clear as possible right from the start.

Reply:
Agreed, I thought this would be the counterargument, and I accept it :) I will be sure to change it when I return to this, before merging.